### PR TITLE
Correlation and Confusion Matrices for Enhanced Model Evaluation #5

### PR DESCRIPTION
**Pull request by Dishant-garg**
solved issue #5 
used various techniques like simple imputer strategy mean and strategy most frequent to handle null values
use label encoder to handle the categorical data
use correlation matrix to visualize the data
cof
after that split data into test set training set
as we get best accuracy in random forest we predict y_test from it 
Screenshot (448)
Screenshot (449)
@Samcoding5854 @darshbaxi pls review the pr

### CODE REVIEW

**Comment by Samcoding5854** [comment]
Good work creating the matrices and evaluation
